model/item.h

```diff
@@ -94,13 +94,29 @@ namespace minsky
     /// mark item on canvas, then throw
     [[noreturn]] void throw_error(const std::string&) const;
 
+     /// @{ current rotation angle associated with this item, update bounding box if rotated
+    virtual double _rotation() const;  
+    virtual double _rotation(double);
+    ecolab::Accessor<double> rotation {
+      [this]() {return _rotation();},
+        [this](double x){return _rotation(x);}};
+    /// @} 
+    
+    
+    Item() {}
+    Item(const Item& x): m_rotation(x.m_rotation) {}
+    Item& operator=(const Item& x) {
```

Comment:
Ditto with assignment, however since Accessor defines assignment, the default compiler generated version suffices.